v3/src/components/graph/hooks/graph-hooks.ts

```diff
@@ -41,34 +43,42 @@ export const useGetData = (props: IUseGetDataProps) => {
     yScale = layout.axisScale("left")
 
   const result = useMemo(() => {
-    let xAttrId= '', yAttrId = '', xName = '', yName = ''
+    let xAttrId = '', yAttrId = '', xName = '', yName = ''
     const data: InternalizedData = {
       xAttributeID: '',
       yAttributeID: '',
       cases: []
     }
 
     const findNumericAttrIds = (attrsToSearch: IAttribute[]) => {
-      for (const iAttr of attrsToSearch) {
-        if (iAttr.type === 'numeric') {
-          if (xAttrId === '') {
-            xAttrId = iAttr.id
-          } else if (yAttrId === '') {
-            yAttrId = iAttr.id
-          } else {
-            break
+        for (const iAttr of attrsToSearch) {
+          if (iAttr.type === 'numeric') {
+            if (xAttrId === '') {
+              xAttrId = iAttr.id
+            } else if (yAttrId === '') {
+              yAttrId = iAttr.id
+            } else {
+              break
+            }
           }
         }
+      },
```

Comment:
Seems to be some unnecessary indenting here.

Reply:
I don't think so. IntelliJ's formatter doesn't either.